### PR TITLE
Update the repo readmes and changelog

### PR DESCRIPTION
**Pull request by RickWinter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,37 +2,36 @@
...
+## Packages available
+Each service might have a number of libraries available from each of the following categories:
+* [Client - New Releases](#Client-New-Releases)
+* [Client - Previous Versions](#Client-Previous-Versions)
 
-## Latest Release
+### Client: New Releases
 
-> NOTE: All code in this repro is in development state and should not be used in production.
+New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview**. These libraries follow the [Azure SDK Design Guidelines for C++](https://azure.github.io/azure-sdk/cpp_introduction.html) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure::Core](sdk/core/azure-core/README.md).
```
Comment (antkmsft) [inline README.md:20]: You are using "preview" word here. Do we need to change it to beta? You know better, just bringing your attention.

File: README.md [hunk 2]
```diff
@@ -2,37 +2,36 @@
 
 [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/cpp/cpp%20-%20client%20-%20ci?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1611&branchName=master)
 
-This repository is for active development of the Azure SDK for C++.
+This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [public developer docs](https://azure.github.io/azure-sdk-for-cpp) or our versioned [developer docs](https://azure.github.io/azure-sdk-for-cpp).
```
Comment (antkmsft) [inline README.md:5]: Links are incorrect, you probably know this, but just bringing your attention in case we actually have these docs published already.

**Review by antkmsft — APPROVED** [review]

**Review by vhvb1989 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RickWinter) [reply README.md:20]: Good catch, meant to use beta.  My brain doesn't want to let go of preview

File: README.md (hunk 2 above)
Comment (RickWinter) [reply README.md:5]: There isn't a correct docs.microsoft.com link for us yet.  We will need to update these links when we get that home setup.  For now both links point to our reference docs

**Pull request merged**